packages/vuetify/src/components/VFileInput/VFileInput.ts

```diff
@@ -115,7 +115,7 @@ export default VTextField.extend({
       return this.isDirty
     },
     isMultiple (): boolean {
-      return this.$attrs.hasOwnProperty('multiple')
+      return this.$attrs.hasOwnProperty('multiple') && this.$attrs.multiple
```

Comment:
Hmm, why are we even checking this through $attrs, and not just `this.multiple`?

Reply:
Yes you are right, maybe confused with an input element.

Reply:
In #9530 I compared the multiple attribute to false because it was an empty string when adding the attribute without a value, which should result in true. The isMultiple should return the correct boolean in these cases:
```html
<v-file-input></v-file-input> // false
<v-file-input multiple></v-file-input> // true
<v-file-input :multiple="false"></v-file-input> // false
<v-file-input :multiple="true"></v-file-input> // true
```

I don't know enough about the Vuetify core to fix this properly, but I also noticed that "multiple" isn't defined in the props either. Isn't this required for it to be processed as a boolean?

Reply:
yes. @nkoik add multiple prop definition and use `this.multiple` instead.